src/utils/achievements.ts

```diff
@@ -0,0 +1,96 @@
+import { Campaign } from 'config/constants/types'
+import { getPointCenterIfoAddress } from 'utils/addressHelpers'
+import { getContract } from 'utils/web3'
+import pointCenterIfo from 'config/abi/pointCenterIfo.json'
+import ifosList from 'config/constants/ifo'
+import { campaignMap } from 'config/constants/campaigns'
+import { Achievement, TranslatableText } from 'state/types'
+import makeBatchRequest from './makeBatchRequest'
+
+interface IfoMapResponse {
+  thresholdToClaim: string
+  campaignId: string
+  numberPoints: string
+}
+
+export const getPointCenterClaimContract = () => {
+  return getContract(pointCenterIfo, getPointCenterIfoAddress())
+}
+
+export const getAchievementTitle = (campaign: Campaign): TranslatableText => {
+  switch (campaign.type) {
+    case 'ifo':
+      return {
+        id: 999,
+        fallback: `IFO Shopper: ${campaign.title}`,
+        data: {
+          name: campaign.title as string,
+        },
+      }
+    default:
+      return campaign.description
+  }
+}
+
+export const getAchievementDescription = (campaign: Campaign): TranslatableText => {
+  switch (campaign.type) {
+    case 'ifo':
+      return {
+        id: 999,
+        fallback: `Committed more than $5 worth of LP in the ${campaign.title} IFO`,
+        data: {
+          name: campaign.title as string,
+        },
+      }
+    default:
+      return campaign.description
+  }
+}
```

Comment:
The idea here is these two functions will handle returning the correct translation for the achievement type.